storybook/stories/Sidebar.stories.js

```diff
@@ -5,21 +5,7 @@ import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import StoryDecorator from './support/StoryDecorator';
 import Sidebar from '../../source/renderer/app/components/sidebar/Sidebar';
-import walletsIcon from '../../source/renderer/app/assets/images/sidebar/wallet-ic.inline.svg';
-import settingsIcon from '../../source/renderer/app/assets/images/sidebar/settings-ic.inline.svg';
-
-const SIDEBAR_CATEGORIES = [
```

Comment:
@yakovkaravelov please restore this part. It is better than the one in the application code.